dashboards/GatherMetrics.json

```diff
@@ -24,10 +24,102 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 5,
+  "id": 22,
   "links": [],
   "liveNow": false,
   "panels": [
+    {
+      "datasource": {
+        "type": "prometheus"
+      },
```

Comment:
it should be (I think) either just string "Prometheus" or an object as you did but the same as in rest of the doc so add here 

"datasource": {
        "type": "prometheus",
        "uid": "000000001"
      },

Reply:
remember that all doc need to be adjusted so I guess it will be easier to adjust to what is already in the doc.

Reply:
alright, so after some reading - grafana used to use "by name" approach for data source (enabling re-importing on various instances _fairly_ simple), right now it's returning uid for provisioned dashboards and since all of existing one have some uid - I assume it's secured on provisioning new instance to re-use uid. Bottom line - the safest approach seems to aling uid with the already existing one :)